CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ set(TTMLIR_ENABLE_BINDINGS_PYTHON ON CACHE BOOL "Enable Python bindings")
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake/modules)
 
-add_compile_options(-Wall -Wextra -Wpedantic -Werror -Wno-unused-parameter --system-header-prefix=/opt/ttmlir-toolchain)
+add_compile_options(-Wall -Wextra -Wpedantic -Wno-unused-parameter --system-header-prefix=/opt/ttmlir-toolchain)
```

Comment:
Not sure if accidental, but we shouldn't remove this flag.

Reply:
The uplifted version of TT-Metal has uint/int comparison warnings in it. Not sure what to do about that.

Reply:
It's passing on main though? Or is it the conv file you included that's causing the issues?

Reply:
Its the conv file I've included yes

Reply:
Gotta call in the big guns here... @nsmithtt please report for duty

Reply:
This cannot land, we definitely want `-Werror`.  Please add to the list of ignored warnings at the top of file `runtime/include/tt/runtime/detail/ttnn.h`. e.g.
```c++
#pragma clang diagnostic ignored "-Wundefined-inline"
```